scripts/install

```diff
@@ -277,17 +282,17 @@ ynh_add_systemd_config --service=$service_name
 
 # Remove IPv6 address set if there is a VPN installed
 if [[ $ip6_addr != none ]]; then
-  if ip -6 address show dev tun0 2> /dev/null | grep -q "${ip6_addr}/"; then
-    ip address delete "${ip6_addr}/128" dev tun0 &> /dev/null
-  fi
+	if ip -6 address show dev tun0 2> /dev/null | grep -q "${ip6_addr}/"; then
+		ip address delete "${ip6_addr}/128" dev tun0 &> /dev/null
+	fi
 fi
 
 #=================================================
 # INTEGRATE SERVICE IN YUNOHOST
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..."
 
-yunohost service add $service_name --description "Creates a Wi-Fi access point" --test_status "systemctl is-active hostapd"
+yunohost service add $service_name --description "Creates a Wi-Fi access point" --test_status "systemctl is-active hostapd" --needs_exposed_ports 67 547
```

Comment:
c.f. https://github.com/labriqueinternet/hotspot_ynh/pull/71#pullrequestreview-622716161

I understand that the linter may be complaining about this, but this is a special case : we don't want to declare that these need to be exposed, because they're only meant for LAN and that would be pretty bad to expose them on the global internet i guess

```suggestion
yunohost service add $service_name --description "Creates a Wi-Fi access point" --test_status "systemctl is-active hostapd"
```